io.gitlab.o20.word.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "io.gitlab.o20.word",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "tags": ["standalone"],
+    "desktop-file-name-suffix": " (Standalone)",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland", "--device=dri", "--filesystem=host"],
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "o20coreapps",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                     "type": "git",
+                     "url": "https://gitlab.com/abstractsoftware/o20/o20coreapps.git",
+                     "branch": "master",
```

Comment:
Flathub only hosts stable applications, this should be a tagged release or a tarball.